### ISSUE TITLE
Custom bees not spawning in world

### ISSUE THREAD

**Issue by bessiq** [Team-Resourceful/ResourcefulBees#20]
**Describe the bug**
I've customized the default bees. I checked their syntax against the (updated) wiki, and made a bee with the new bee creator web page to compare against also. (The web page puts numerical values like the size modifier and output weights in quotes, fwiw - doesn't resolve my issue regardless.) I've made a test profile with no other mods except JEI, TOP, and other client-side GUI mods - along with this mod. Still no bees. I removed biome restrictions for many of them, just to have a better chance of seeing them spawn...somewhere. Nada.

If I leave every config in this mod untouched, I get the default bees all day long. But the whole point of the mod is to make custom bees, yeah? But when I do that, I never see the custom bees spawn in the world. No nests. No bees. Nothing. I see the logs saying "X bee passed validation check!" for all my bees. But where are they? I can spawn them manually from the "egg" item. I can see their honeycombs and honeycomb blocks in JEI and hold/place them. I didn't lower the global spawn weight. I triple-checked that spawn in world equals true. I'm at a loss.

Here's my latest stripped-down version of my custom bee configs, including the latest log:

https://www.dropbox.com/sh/vja1mcq7mqdmzid/AAButbX3d6pS7HMTpZ8qqlBza?dl=0

**To Reproduce**
1. make custom bee(s)
2. load world
3. look for said bee(s)
4. give up

**Expected behavior**
bees spawn naturally, as nature intended

**Screenshots**
n/a

**please complete the following information:**
 - Forge Version 1.15.2 - 31.2.9
 - Mod Version 0.2.4a

**Additional context**
bees

Thanks.

**Comment by Epic428** [comment]
Some bees you've specified as `"spawnInWorld": true` but never specified the biomes for where they could spawn, the ones where you did specify the biomes where they could spawn you used the outdated `biomeList`, vs the newer `biomeWhitelist`/`biomeBlacklist`. We made the biome spawning more customizable with a whitelist and blacklist feature.

**Comment by Epic428** [comment]
Also wiki has updated info on using the whitelist/blacklist feature, but keep in mind white list is required if you want them to spawn, blacklist is more helpful when dealing with tags but still useful. 

edit:I should also note that simply because of the `biomeList` vs `biomeWhitelist` that is why the bee passed validation but still they did not spawn. it just assumed they had no valid spawns.

**Comment by bessiq** [comment]
Okay, sorry I did not notice the updated biome wording. And I deleted the biome lists for several of the bees mainly as a test, assuming that would allow them to spawn anywhere. But now I see I am required to define the biomes (using the new whitelist/blacklist) if I set spawn in world to true. Thanks for clearing that up.

And do the quotes around the numerical values matter? e.g. for output weights, size, etc.
